cluster-up/cluster/k8s-1.13.3/provider.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="k8s-1.13.3@sha256:9c2b78e11c25b3fd0b24b0ed684a112052dff03eee4ca4bdcc4f3168f9a14396"
+
+source ${KUBEVIRTCI_PATH}/cluster/ephemeral-provider-common.sh
+
+function up() {
+    ${_cli} run $(_add_common_params)
```

Comment:
at least for k8s provider it a lot of duplication under the `up` function, maybe we can somehow to reuse it

Reply:
yep, i agree. I'd like to follow up with this to reduce the scope of the changes in this pr though.